### PR TITLE
[KEP-3017]: promote PDBUnhealthyPodEvictionPolicy to GA

### PR DESCRIPTION
**Pull request by atiratree**
add docs for PDBUnhealthyPodEvictionPolicy GA promotion

- https://github.com/kubernetes/kubernetes/pull/123428
- https://github.com/kubernetes/enhancements/issues/3017

### CODE REVIEW

**Review by celestehorgan — APPROVED** [review]

**Comment by sftim** [comment]
This looks like a draft
/approve cancel

**Comment by reylejano** [comment]
Should this target the `dev-1.30` branch
/assign @drewhagen

**Comment by drewhagen** [comment]
It looks like this document is associated with a KEP that was removed from the 1.30 milestone.

It should probably be targeted at `dev-1.31` branch in the next cycle.

**Comment by drewhagen** [comment]
Heads up that `dev-1.31` branch is created now.

**Comment by Princesso** [comment]
Hello @atiratree :wave: please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday July 16th 2024 18:00 PST. Thank you!

**Comment by Princesso** [comment]
Hi @atiratree, can you move this PR out of Draft, if it is ready for review. Thank you!

**Comment by Princesso** [comment]
Hi @atiratree, a gentle reminder that tomorrow is the deadline for having your Docs PR ready for review. Please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before tomorrow, Tuesday, July 16th, 2024 18:00 PST.

**Comment by salaxander** [comment]
Hi @atiratree, reminder that today is the deadline for having your Docs PR ready for review. Please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review today at 18:00 PST.

**Comment by atiratree** [comment]
I am sorry for the delay, the PR is ready for review.

File: content/en/docs/reference/command-line-tools-reference/feature-gates/pdb-unhealthy-pod-eviction-policy.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.27"
+    toVersion: "1.29"
```
Comment (dipesh-rawat) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/pdb-unhealthy-pod-eviction-policy.md:16]: The `toVersion` needs correction.

```suggestion
    toVersion: "1.30"
```

**Review by soltysh — APPROVED** [review]
Minor nit from Dipesh, but otherwise this lgtm from sig-apps pov.

File: content/en/docs/reference/command-line-tools-reference/feature-gates/pdb-unhealthy-pod-eviction-policy.md (hunk 1 above)
Comment (atiratree) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/pdb-unhealthy-pod-eviction-policy.md]: ouch, thx a lot for catching that

**Comment by atiratree** [comment]
Thanks, the technical review was done already: https://github.com/kubernetes/website/pull/45263#pullrequestreview-2187934690

**Comment by tengqm** [comment]
Docs looks good. Upstream PR merged.
/approve

**Pull request merged**